tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -262,6 +297,18 @@ Object.PCM {
 			formats	'S16_LE,S24_LE,S32_LE'
 		}
 	}
+
+	pcm."27" {
```

Comment:
pcm.2 please. Lets keep it incremental as it is really only an instance ID.

Reply:
Here I followed https://github.com/thesofproject/sof/pull/6783#issue-1493569730

Reply:
@yongzhi1 the 27 is set below in the ID attribute. What I mean was the instance ID here doesnt need to match it